bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/sitequality/SiteQualityPanel.java

```diff
@@ -250,38 +256,59 @@ public void updateEnabledState(boolean enabled) {
         elevMin.setEnabled(e);
         elevMax.setEnabled(e);
     }
-}
 
+	private class GBC extends GridBagConstraints {
+		GBC(int gridx, int gridy) {
+			this.gridx = gridx;
+			this.gridy = gridy;
+			insets = new Insets(0, 3, 1, 3);
+			fill = HORIZONTAL;
+			anchor = EAST;
+		}
 
+		GBC(int gridx, int gridy, Insets insets) {
+			this(gridx, gridy, 1, 1, insets);
+		}
 
+		GBC(int gridx, int gridy, int xspan, int yspan, Insets insets) {
+			this(gridx, gridy);
+			this.gridwidth = xspan;
+			this.gridheight = yspan;
+			Insets prev = this.insets;
+			this.insets = new Insets(
+					prev.top + insets.top,
+					prev.left + insets.left,
+					prev.bottom + insets.bottom,
+					prev.right + insets.right);
+		}
+	}
+}
 
 
-@SuppressWarnings("serial")
 class TimingWindowTableModel extends DefaultTableModel implements PropertyChangeListener {
-
-	enum Cols {
+	private enum Cols {
 		Window, Duration, Repeats, Period
 	}
 
 	private static final long MS_PER_SECOND = 1000;
 	private static final long MS_PER_MINUTE = MS_PER_SECOND * 60;
 	private static final long MS_PER_HOUR = MS_PER_MINUTE * 60;
 
-    private static final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss z");
-    static {
-    	dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
-    }
+    private static final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss z") {{
```

Comment:
I'm not sure how `DefaultTableModel`s work in terms of thread safety, but I assume they would be thread-safe. No reason why this can't be changed to a `DateFormatter` instead, though. I'll go ahead and do that: it should be straightforward.